package/kedro_viz/server.py

```diff
@@ -213,12 +213,27 @@ def commands():
     type=click.Path(dir_okay=False, writable=True),
     help="Path to save the pipeline JSON file",
 )
-def viz(host, port, browser, load_file, save_file):
+@click.option(
+    "--pipeline",
+    type=str,
+    default=None,
+    help="Name of the modular pipeline to run."
+    "If not set, the default pipeline is visualized",
+)
+def viz(host, port, browser, load_file, save_file, pipeline):
     """Visualize the pipeline using kedroviz."""
-    _call_viz(host, port, browser, load_file, save_file)
-
-
-def _call_viz(host=None, port=None, browser=None, load_file=None, save_file=None):
+    _call_viz(host, port, browser, load_file, save_file, pipeline)
+
+
+# pylint: disable=too-many-arguments
+def _call_viz(
+    host=None,
+    port=None,
+    browser=None,
+    load_file=None,
+    save_file=None,
+    pipeline_name=None,
+):
     global data  # pylint: disable=global-statement,invalid-name
 
     if load_file:
```

Comment:
I think the whole section can be refactored into a separate function (like `data = _load_from_file(load_file)`)